packages/terra-clinical-item-collection/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Unreleased
 4.19.0 - (April 14, 2020)
 ----------
 ### Changed
+* Jest and wdio snapshot updated for ItemDisplay's primary textStyle default prop.
```

Comment:
this need to be updated to the unreleased section

Reply:
Moved 87f5525